app/pb_kits/playbook/pb_title/_title.jsx

```diff
@@ -7,14 +7,15 @@ import { spacing } from '../utilities/spacing.js'
 
 type TitleProps = {
   aria?: object,
-  className?: String,
   children?: Array<React.ReactNode> | React.ReactNode,
+  className?: String,
   dark?: Boolean,
   data?: object,
   id?: String,
   size?: 1 | 2 | 3 | 4,
   tag?: "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "div",
   text?: String,
+  variant?: Boolean,
```

Comment:
I think the title should be clickable whenever the color blue is applied. What do you think about changing the prop to "link" instead of "variant"?

Reply:
Thanks for the comment. I will ask for clarification today at daily.

Reply:
@kellyeryan Nevermind, as what the team has discussed at Dailies.